docs/usage.md

```diff
@@ -119,6 +119,22 @@ To commit all the tracked changes in base repo and any enabled modules:
 evg-module-manager git-commit --commit-message "my commit message"
 ```
 
+## Creating github pull request
+
+Use `pull-request` command to create pull requests across base repo and all enabled modules.
+In order to fully utilize this function, [github CLI](https://cli.github.com/) need to be installed
```

Comment:
What is meant by "fully utilize"? Is there some partial functionality that can be used without the github cli?

Reply:
I'm trying to emphasis that this function is relying on github CLI, otherwise user cannot use this function. Rephrased this line.